shell/platform/embedder/embedder.h

```diff
@@ -449,6 +459,13 @@ typedef uint32_t (*UIntFrameInfoCallback)(
     void* /* user data */,
     const FlutterFrameInfo* /* frame info */);
 
+/// Callback for when a frame buffer object is requested with necessary
+/// information for partial repaint.
+typedef void (*FlutterFrameBufferWithDamageCallback)(
+    void* /* user data */,
+    const intptr_t id,
```

Comment:
nit: consider calling this `fbo_id`. Just `id` lacks the context.

Reply:
I just realized that there's actually no need to name the arguments at this point. I updated them as comments though.